src/core/CoolBoard.h

```diff
@@ -77,13 +77,15 @@ class CoolBoard {
   bool mqttsConfig();
   static int b64decode(String b64Text, uint8_t *output);
   void mqttsConvert(String cert);
-  void updateFirmware(String firmwareVersion, String firmwareUrl, String firmwareUrlFingerprint);
+  void updateFirmware(String firmwareVersion, String firmwareUrl,
+                      String firmwareUrlFingerprint);
   void tryFirmwareUpdate();
 
 private:
   CoolBoardSensors coolBoardSensors;
   CoolBoardLed led;
-  CoolTime rtc;
+  CoolTime coolTime;
+  DS1337 rtc;
```

Comment:
why do you need an instance of RTC here, if you already have one in `CoolTime` ? This is confusing !